build-support/bin/check_pants_pex_abi.py

```diff
@@ -39,28 +36,28 @@ def main():
           .format(', '.join(sorted(parsed_abis))))
```

Comment:
I did not use f strings here, nor two lines above, because I found it less readable to inline the `', '.join(sorted(parsed_abis))`.

Alternative approach is to introduce intermediate variables.

```python
expeccted_abis = ', '.join(sorted(parsed_abis))
success(f"Success. The pants.pex was built with wheels carrying the expected ABIs: {expected_abis}."
```